_posts/en/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,264 @@
+    e.g.  `thresh(3,after(1),after(500000001),pk(A))`.  Note: as of this
+    writing, this change is only a [pending PR][rust-miniscript #121] to
+    the Rust version of the miniscript library and has not yet
+    propagated to the online live demos for [miniscript][miniscript demo] or [minsc][].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Swan supports sending to bech32 addresses:**
+  Swan, through their custodian Prime Trust, now [supports withdrawals][swan bech32 send]
+  to [bech32][topic bech32] addresses.
+
+- **Ledger adds manual coin selection support:**
+  Moving from a first-in, first-out UTXO selection model, Ledger now also
+  supports [manual coin selection][ledger coin control article] for either privacy
+  or fee-minimization benefits.
+
+- **Sparrow wallet announced:**
+  [Sparrow wallet][sparrow twitter thread] is a new desktop wallet supporting
+  single or multisignature addresses, PSBT, hardware wallets, and [coin
+  selection][topic coin selection].
+
+- **JoinMarket 0.7.0 adds BIP78, PSBT:**
+  [JoinMarket 0.7.0][joinmarket 0.7.0] includes support for [BIP78 payjoin][topic payjoin]
+  for payments. PSBT support was also added in order to facilitate the implementation.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta.rc4][lnd 0.11.1-beta] is the release candidate for a
+  minor version.  It's release notes summarize its changes as, "a number
+  of reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor commitments][topic
+  anchor outputs] spec compliant."
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16378][] adds a new `send` RPC method to the wallet. This new
+  RPC method is designed for maximum flexibility and includes options such as
+  multiple outputs (allowing [payment batching][]), [coin selection][topic coin
+  selection], manual or automatic feerates, and [PSBT format][topic psbt]. It is
+  intended to unify the functionality of the `sendtoaddress` and `sendmany` RPC
+  methods, which may be deprecated in a future release. For the full list of
+  options see the [RPC help text][send rpc help].
+
+- [Bitcoin Core #19643][] adds a new `-netinfo` option to the `bitcoin-cli`
+  command which displays a peer connection dashboard. This dashboard offers
```

Comment:
"which" here needs a comma before it, but probably s/which displays/to display/ is better